src/puma-cli.js

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env node
+'use strict';
+const program = require('commander');
+const puma = require('pumascript');
+const fs = require("fs");
+const injector = require('./injector');
+
+
+
+let run = (req, op) => {
+    puma.evalPuma(fs.readFileSync(req, 'Utf8'));
+}
+
+
+let compile = (req, op) => {
+    if (req.split('.')[1] !== 'puma') {
+        console.log('this file not is pumascript file');
```

Comment:
Is this a warning?... `console.warn(...` but else block would not execute so
i guess script should fail... when errors are encountered please add `process.exit(error level 1 or greater)` so process signals non-completion and exits with appropriate errorlevel